webhooks/README.md

```diff
@@ -53,14 +53,84 @@ function isSignatureValid(fidelHeaders, payload, secret, url) {
 ```
 
 To prevent replay attacks where a valid payload and its signature is intercepted and re-transmitted, you can use the `x-fidel-timestamp` header and confirm that the timestamp is not too old. We recommend you validate the requests in a 5 minute gap. In case of retries, a new signature and timestamp are generated for each new request.
+---
+
+## Custom headers
+Fidel allows you to define custom HTTP headers that are passed in the webhook POST requests sent to you application.
+Custom headers can be defined when creating a new webhook in the dashboard or by using the [Webhooks API](https://reference.fidel.uk/v1/reference#create-webhook-brand) by setting the optional `headers` object with a key-value pair. You can use the [Update  Webhook](https://reference.fidel.uk/reference#create-webhook-program) endpoint and submit an empty `headers` object in order to remove the custom headers from the webhook.
+
+See below an example on how to create a webhook with a custom header using the [Create Webhook](https://reference.fidel.uk/reference#create-webhook-program) endpoint:
+
+```sh
+curl -X POST \
+  https://api.fidel.uk/v1/programs/06471dbe-a3c7-429e-8a18-16dc97e5cf35/hooks \
+  -H 'Content-Type: application/json' \
+  -H 'Fidel-Key: sk_test_50ea90b6-2a3b-4a56-814d-1bc592ba4d63' \
+  -d '{
+        "event": "transaction.auth",
+        "url": "https://realtime.com",
+        "headers": {
+  	       "Custom-Header": "my-custom-header"
+        }
+      }'
+```
+
+Up to 5 custom headers per webhook can be defined and they need to follow strict character validation patterns. You can find more information regarding validation in the [Webhooks API](https://reference.fidel.uk/v1/reference#create-webhook-brand) reference.
+
+Additionally, the HTTP reserved headers are blacklisted and cannot be used. You can see the full list of blacklisted header names below:
 
+```json
+[
+  "Accept",
+  "Accept-Charset",
+  "Accept-Encoding",
+  "Accept-Language",
+  "Accept-Datetime",
+  "Access-Control-Request-Method",
+  "Access-Control-Request-Headers",
+  "Cache-Control",
+  "Connection",
+  "Content-Length",
+  "Content-Type",
+  "Cookie",
+  "Date",
+  "Expect",
+  "Forwarded",
+  "From",
+  "Host",
+  "If-Match",
+  "If-Modified-Since",
+  "If-None-Match",
+  "If-Range",
+  "If-Unmodified-Since",
+  "Max-Forwards",
+  "Origin",
+  "Pragma",
+  "Proxy-Authorization",
+  "Range",
+  "Referer",
+  "Te",
+  "Transfer-Encoding",
+  "User-Agent",
+  "Upgrade",
+  "Via",
+  "Warning",
+  "Fidel-Account",
+  "Fidel-Key",
+  "Fidel-Live",
+  "Fidel-Request-Id",
+  "X-Fidel-Signature",
+  "X-Fidel-Timestamp",
+  "Fidel-User"
+]
+```
 ---
 
+
 ## Events
 
 ### Brand
-
```

Comment:
removing these newlines will do nothing on the frontend, we'll just have to fix margins and we already have a card for that – but they do no harm, just fyi.